recipe/meta.yaml

```diff
@@ -11,47 +11,68 @@ source:
   sha256: 4fd2eb3736cea503be94ab18d31a669addbaf2dd6a8249527cef94163bb93975
 
 build:
+  number: 4
   noarch: python
-  number: 3
-  script: python -m pip install . -vv
 
-requirements:
-  host:
-    - hatchling
-    - pip
-    - python {{ python_min }}
+outputs:
+  - name: osmnx-base
+    script: build_base.sh
+    build:
+      noarch: python
```

Comment:
Isn't `noarch` defined on line 15 already?

Reply:
It appears that both need to be defined, otherwise it will not consider the build to be `noarch`. I tried without this first but realized that it needs to be defined.

Reply:
Is this still the case now that the multiple outputs have been approved?

Reply:
Yes, the issue caused by this was different from the approval one. Now, each output has its own `noarch` block.